datasets/shear_flow/README.md

```diff
@@ -75,15 +75,15 @@ The initial condition is thus indexed by $n_\text{shear},n_\text{blobs},w$.
 
 **Approximate time to generate the data:** per input parameter: $\sim 5h$, total: $\sim 25$ hours.
 
-**Hardware used to generate the data and precision used for generating the data:** 512 nodes of 96 CPU cores each with 96 tasks running in parallel on each node, in single precision.
+**Hardware used to generate the data and precision used for generating the data:** 512 nodes of 96 CPU cores each with 96 tasks running in parallel, in single precision.
```

Comment:
Your code is double precision for the sim itself, isn't it?

Reply:
right